src/pages/WorkspaceSwitcherPage.tsx

```diff
@@ -242,12 +242,9 @@ function WorkspaceSwitcherPage({policies}: WorkspaceSwitcherPageProps) {
                         onSelectRow={selectPolicy}
                         shouldPreventDefaultFocusOnSelectRow
                         headerMessage={headerMessage}
-                        shouldShowTooltips
-                        autoFocus={false}
```

Comment:
This should be implemented

Reply:
you mean `autoFocus`? Or something more?

Reply:
Yes. Currently the input is always auto focused. Add this prop and if it's false return early

https://github.com/Expensify/App/blob/a4e9804e1a704a3412149850f685d14a41f5021e/src/components/SelectionList/BaseSelectionList.tsx#L378-L395

Reply:
Done, but I am a bit concerned, autoFocus was set to false previously, but on staging the text input get auto focus...

Reply:
Sorry actually the `WorkspaceSwitcherPage` should keep the auto focus functionality. But it should be done only from one place. Currently we are using `useAutoFocusInput` hook to do so that's why we need to pass `autoFocus={false}` so it's not autofocused by the other component too.

Reply:
Got it, done